frontend/src/components/search/SearchResultsListItem.tsx

```diff
@@ -0,0 +1,129 @@
+"use client";
+
+import { Opportunity } from "src/types/search/searchResponseTypes";
+import { formatDate } from "../../utils/dateUtil";
+
+interface SearchResultsListItemProps {
+  opportunity: Opportunity;
+}
+
+export default function SearchResultsListItem({
+  opportunity,
+}: SearchResultsListItemProps) {
+  const opportunityURL =
+    process.env.NEXT_PUBLIC_ENVIRONMENT === "prod"
+      ? "https://grants.gov"
+      : "https://test.grants.gov";
+
+  const metadataBorderClasses = `
+    display-block
+    tablet:display-inline-block
+    tablet:border-left-1px
+    tablet:padding-x-1
+    tablet:margin-left-neg-1
+    tablet:margin-right-1
+    tablet:border-base-lighter
+  `;
+
+  return (
+    <div
+      className="
+        border-1px
+        border-base-lighter
+        padding-x-2
+        padding-y-105
+        margin-bottom-2
+        text-base-darker
+      "
+    >
+      <div className="grid-row grid-gap">
+        <div className="desktop:grid-col-fill">
+          <div className="grid-row flex-column">
+            <div className="grid-col tablet:order-2">
+              <h2 className="margin-y-105 line-height-serif-2">
+                <a
+                  href={`${opportunityURL}/search-results-detail/${opportunity?.opportunity_id}`}
+                  className="usa-link usa-link--external"
+                >
+                  {opportunity?.opportunity_title}
+                </a>
+              </h2>
+            </div>
+            <div className="grid-col tablet:order-1 overflow-hidden font-body-xs">
+              {opportunity.opportunity_status === "archived" && (
+                <span className={metadataBorderClasses}>
+                  <strong>Archived:</strong>{" "}
+                  {opportunity?.summary?.archive_date
+                    ? formatDate(opportunity?.summary?.archive_date)
+                    : "--"}
+                </span>
+              )}
+              {(opportunity?.opportunity_status === "archived" ||
+                opportunity?.opportunity_status === "closed") &&
+                opportunity?.summary?.close_date && (
+                  <span className={metadataBorderClasses}>
+                    <strong>Closed:</strong>{" "}
+                    {opportunity?.summary?.close_date
+                      ? formatDate(opportunity?.summary?.close_date)
+                      : "--"}
+                  </span>
+                )}
+              {opportunity?.opportunity_status === "posted" && (
```

Comment:
nit: you might try getPosted() getArchived() functions that return the JSX. And/or a javascript switch statement to clean some of this up. Just a nit though